### DISCUSSION TITLE
Email client reader view

### DISCUSSION THREAD

**Discussion by M-J-Robbins** [email-markup-consortium/email-markup-consortium#92] [Ideas]
## Project title

Email client reader view
## Project description

A browser plug in that enables reader view for webmail clients

## Why is this needed

It's an accessibility tool designed to benefit email recipients.  
It could also help highlight issues with poor markup and help senders improve there.
I think this would be better done directly in the email clients, this might help inspire them to build their own versions of it.

## Scope

Initially, this only covers webmail clients opens in Chrome.
As other browsers support manifest 3, this might expand the support to those browsers.  Or we could look at building a version with manifest 2.
I'd also be interested it trying to build a version that works as a thunderbird plug in.

## Contributing members

@M-J-Robbins

## Where will the project be worked on, and where will the finished project be hosted?

I've already built a working version here https://github.com/M-J-Robbins/email-reader-view
We could transfer that repo to sit inside the EMC GitHub

**Comment by M-J-Robbins** [comment]
A couple of screenshots to show what it looks like with some different settings applied
<div style="display:flex">
<img src="https://user-images.githubusercontent.com/7871206/192007744-66f23312-3bd5-4294-8faa-92b0442e4068.png" width=33% alt="Action rocket Email weekly newsletter">
<img src="https://user-images.githubusercontent.com/7871206/192007860-86e5014e-15b9-461c-80da-41b96fb41832.png" width=33% alt="Action rocket Email weekly newsletter with basic reader view styles">
<img src="https://user-images.githubusercontent.com/7871206/192008044-c41dfd49-ae3c-4719-95b2-85dc640c162a.png" width=33% alt="Action rocket Email weekly newsletter with dark styles">
</div>

**Comment by alicemeichi** [comment]
Approved -- this is a useful idea to help developers identify how to properly structure their markup and as a potential feature email clients can add to aid their users' reading preferences in the interest of Accessibility

**Reply by husseinalhammad** [reply]
If this assists developers in some way, great. But I think the main benefits here go to end users and they should remain the primary audience for this. And the terms used in the extension's settings should be less technical than CSS property names and their literal values as seen in the screenshots.

Unless this extension has an option to work outside of email clients, I think the benefits to developers is minimal. If a developer needs to send themselves a test email just to be able to review how their email looks without CSS and images, they may need to review their workflow too.

**Reply by M-J-Robbins** [reply]
I'm very open to suggestions on naming, I went for technical names, 'cos that's what I know but I would rather this appealed more to users.

**Comment by M-J-Robbins** [comment]
The project has now been moved to the email-markup-consortium github https://github.com/email-markup-consortium/email-reader-view
